src/style.css

```diff
@@ -1,3 +1,61 @@
+:root {
+  --link-color: #00ACDC;
+  --header-height: 60px
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
The 'font-family' declaration should include a fallback font, such as 'sans-serif', in case the primary font is not available. For example, 'font-family: Roboto, Arial, sans-serif;'.